crates/uv/src/commands/help.rs

```diff
@@ -67,11 +67,11 @@ pub(crate) fn help(query: &[String], printer: Printer) -> Result<ExitStatus> {
     };
 
     let is_terminal = std::io::stdout().is_terminal();
-    if !is_root && is_terminal && which("less").is_ok() {
+    if !no_pager && !is_root && is_terminal && which("less").is_ok() {
```

Comment:
Maybe we should collapse these into a `should_page` bool haha it's getting kind of long.

Reply:
Will do! 👍